src/routes/console/project-[project]/auth/security/updateMockNumbers.svelte

```diff
@@ -0,0 +1,128 @@
+<script lang="ts">
+    import { Submit, trackError, trackEvent } from '$lib/actions/analytics';
+    import { CardGrid, Heading } from '$lib/components';
+    import { InputPhone, InputText } from '$lib/elements/forms';
+    import { Button, Form, FormItem, FormItemPart } from '$lib/elements/forms';
+    import type { MockNumber } from '$lib/sdk/auth';
+    import { sdk } from '$lib/stores/sdk';
+    import { project } from '../../store';
+    import { addNotification } from '$lib/stores/notifications';
+    import { invalidate } from '$app/navigation';
+    import { Dependencies } from '$lib/constants';
+    import { writable } from 'svelte/store';
+    import Empty from '$lib/components/empty.svelte';
+
+    let numbers = writable<MockNumber[]>($project.authMockNumbers);
+    let initialNumbers = [];
+    let projectId: string = $project.$id;
+    let submitDisabled = true;
+
+    $: initialNumbers = $project.authMockNumbers.map((num) => ({ ...num }));
+    $: submitDisabled = JSON.stringify($numbers) === JSON.stringify(initialNumbers);
```

Comment:
we might have some helper function for this in `$lib/utils`.

This approach depends on order too

Reply:
Are you sure its lib/utils ? I can't seem to find such a folder.

Reply:
nvm, those where for arrays